extensions/amp-analytics/0.1/test/test-analytics-root.js

```diff
@@ -333,25 +333,108 @@ describes.realWin('AmpdocAnalyticsRoot', {amp: 1}, env => {
       addTestInstance(root3.getElement(body, '#target'), null);
     });
 
-    it('should find an AMP element for AMP search', () => {
+    it('should find an AMP element for AMP search', async () => {
       child.classList.add('i-amphtml-element');
-      return root.getAmpElement(body, '#child').then(element => {
-        expect(element).to.equal(child);
-      });
+      const elements = await root.getAmpElementOrElements(body, '#child');
+      expect(elements[0]).to.equal(child);
     });
 
-    it('should allow not-found element for AMP search', () => {
-      return root.getAmpElement(body, '#unknown').catch(error => {
+    it('should allow not-found element for AMP search', async () => {
+      await root.getAmpElementOrElements(body, '#unknown').catch(error => {
         expect(error).to.match(/Element "#unknown" not found/);
       });
     });
 
-    it('should fail if the found element is not AMP for AMP search', () => {
+    it('should fail if the found element is not AMP for AMP search', async () => {
       child.classList.remove('i-amphtml-element');
-      return root.getAmpElement(body, '#child').catch(error => {
+      await root.getAmpElementOrElements(body, '#child').catch(error => {
         expect(error).to.match(/required to be an AMP element/);
       });
     });
+
+    describe('get amp elements', () => {
+      let child2;
+      let elements;
+      let error;
+
+      beforeEach(() => {
+        error = false;
+        child2 = win.document.createElement('child');
+        body.appendChild(child2);
+        child.classList.add('i-amphtml-element');
+        child2.classList.add('i-amphtml-element');
+      });
+
+      afterEach(() => {
+        if (!error) {
+          expect(elements).to.contain(child);
+          expect(elements).to.contain(child2);
+          expect(elements.length).to.equal(2);
+        }
+      });
+
+      it('should find elements by ID', async () => {
```

Comment:
This is basically testing the browser provided `querySelectorAll` API. I don't think we need to test browser API.

Reply:
Sg, fixed tests to reflect this.